docs/philosophy.rst

```diff
@@ -108,7 +108,7 @@ by *behave* as well as many other tools.
     then I should have three blue garments in stock
      and two black garments in stock.
 
-Each scenario is an exemplar, designed to illustrate a specific aspect of behavior of the application.
+Each scenario is an exemple, designed to illustrate a specific aspect of behavior of the application.
```

Comment:
That's actually worse (see wiktionary). An "exemplar" could fit, an "ex**a**mple" also.

Reply:
Thank's for the feedback.
I've reverted **exemple** back to **exemplar** in a later commit